openprescribing/media/js/package.json

```diff
@@ -44,6 +44,9 @@
       "jquery": true,
       "browser": true,
       "mocha": true
+    },
+    "rules": {
+      "no-var": 0
```

Comment:
Why's there duplication between this and `eslintrc.json`?

Reply:
This version is for your own eslint which you would install using npm per the README.  The other version is for travis per .travis.yml.